frontend/packages/design-system/src/themes/Link.ts

```diff
@@ -0,0 +1,17 @@
+export default {
+  defaultProps: {
+    isUnderlined: false,
```

Comment:
this doesn't seem to work

Reply:
This works for me. Did you build the design system?